hyundai_kia_connect_api/KiaUvoApiEU.py

```diff
@@ -939,35 +939,79 @@ def charge_port_action(
     def start_climate(
         self, token: Token, vehicle: Vehicle, options: ClimateRequestOptions
     ) -> str:
-        url = self.SPA_API_URL + "vehicles/" + vehicle.id + "/control/temperature"
+        if not vehicle.ccu_ccs2_protocol_support:
+            url = self.SPA_API_URL + "vehicles/" + vehicle.id + "/control/temperature"
+
+            # Defaults are located here to be region specific
+
+            if options.set_temp is None:
+                options.set_temp = 21
+            if options.duration is None:
+                options.duration = 5
+            if options.defrost is None:
+                options.defrost = False
+            if options.climate is None:
+                options.climate = True
+            if options.heating is None:
+                options.heating = 0
+
+            hex_set_temp = get_index_into_hex_temp(
+                self.temperature_range.index(options.set_temp)
+            )
 
-        # Defaults are located here to be region specific
+            payload = {
+                "action": "start",
+                "hvacType": 0,
+                "options": {
+                    "defrost": options.defrost,
+                    "heating1": int(options.heating),
+                },
+                "tempCode": hex_set_temp,
+                "unit": "C",
+            }
+        else:
+            url = (
+                self.SPA_API_URL_V2
+                + "vehicles/"
+                + vehicle.id
+                + "/ccs2/control/temperature"
+            )
 
-        if options.set_temp is None:
-            options.set_temp = 21
-        if options.duration is None:
-            options.duration = 5
-        if options.defrost is None:
-            options.defrost = False
-        if options.climate is None:
-            options.climate = True
-        if options.heating is None:
-            options.heating = 0
+            # Defaults are located here to be region specific
+
+            if options.set_temp is None:
+                options.set_temp = 21
+            if options.duration is None:
+                options.duration = 5
+            if options.defrost is None:
+                options.defrost = False
+            if options.climate is None:
+                options.climate = True
+            if options.heating is None:
+                options.heating = 0
+
+            hex_set_temp = get_index_into_hex_temp(
+                self.temperature_range.index(options.set_temp)
+            )
 
-        hex_set_temp = get_index_into_hex_temp(
-            self.temperature_range.index(options.set_temp)
-        )
+            payload = {
+                "drvSeatLoc": "R",
+                "tempUnit": "C",
+                "seatClimateInfo": {
+                    "rrSeatClimateState": 6,
```

Comment:
Right now these are hard coded to confirm this code even runs.  Once that works we will change this.